src/data/markdown/docs/02 javascript api/01 Init context/open.md

```diff
@@ -21,6 +21,10 @@ By restricting it to the init context, we can easily determine what local files
 
 See the example further down on this page. For a more in-depth description, see [Running k6](/getting-started/running-k6).
 
+#### Breaking change in v0.36.0
+
+Since k6 v0.36.0 we also restricted usage of the `open(...)` under the `__VU` condition. A code like `if (__VU >0) { const arr = open("./arr.json"); }` should cause an error.
```

Comment:
```suggestion
Since k6 v0.36.0 `open(...)` is now restricted to the init context of the VU that's initialized to get the options from the script (`__VU==0`). This means that the code like `if (__VU >0) { const arr = open("./arr.json"); }` will result in error.
```
Trying to improve formulation with rather shameless copy-pasting from https://github.com/grafana/k6/pull/2340 :sweat_smile: But if formulation was done in a satisfying manner there, I think we can re-use it here.

Reply:
```suggestion
Since k6 v0.36.0, VUs are now restricted to only `open()` files that were also opened in the init context of the first VU - the one that was initialized to get the exported `options` from the JS script (`__VU==0`). This means that the code like `if (__VU > 0) { const arr = open("./arr.json"); }` will result in an error.
```